src/scope.ml

```diff
@@ -130,6 +197,35 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
     let c = Pervasives.ref (-1) in
     fun () -> Pervasives.incr c; Printf.sprintf "#%i" Pervasives.(!c)
   in
+  let add_implicits_outermost : p_term -> p_term = fun t ->
+    match t.elt with
+      | (P_Appl(_, _) as e)
+      | (P_Iden(_, _) as e)   ->
```

Comment:
[as e] is useless here since it is equal to t.elt

Reply:
Fixed, thanks.